### ISSUE TITLE
[Question] i can't seem to make analytics work

### ISSUE THREAD

**Issue by adigiz** [urdadx/librelinks#58]
First of all, Great work! this is a really good project. But i just can't seem to make the analytics work. I now encountered an issue 

```
data: {
      error: "The pipe 'libre_device_tracking' does not have an endpoint yet",
      documentation: 'https://docs.tinybird.co/api-reference/pipe-api.html'
    }
```
and i dont understand, do we need to create a data source in tinybird also? or maybe can someone guide me with this analytics?

Thank you

**Comment by Golgrax** [comment]
for data traffics i think.

**Comment by CaseoJKL** [comment]
> First of all, Great work! this is a really good project. But i just can't seem to make the analytics work. I now encountered an issue
> 
> ```
> data: {
>       error: "The pipe 'libre_device_tracking' does not have an endpoint yet",
>       documentation: 'https://docs.tinybird.co/api-reference/pipe-api.html'
>     }
> ```
> 
> and i dont understand, do we need to create a data source in tinybird also? or maybe can someone guide me with this analytics?
> 
> Thank you

Yes we do need to create datasources on Tinybird considering I'm having this error right now `The pipe 'libre_device_tracking' does not exist`
But there isn't enough documentation on this repository to set that up properly.

**Comment by CaseoJKL** [comment]
@urdadx
